### PR TITLE
Feat/dynamic background

### PR DESCRIPTION
**Pull request by luiscovelo**
Hi,

Following ROADMAP, i implemented the dynamic background for mosaic, basically it's was necessary to add a new field into Mosaic struct `backgroundUrl` and change in ffmpeg command.

Best reguards.

### CODE REVIEW

File: internal/mosaic/content.go [hunk 1]
```diff
@@ -17,9 +17,10 @@ type (
 	}
 
 	Mosaic struct {
-		Name      string  `json:"name"`
-		Medias    []Media `json:"medias"`
-		WithAudio bool    `json:"with_audio"`
+		Name          string  `json:"name"`
+		BackgroundUrl string  `json:"backgroundUrl"`
```
Comment (mauricioabreu) [inline internal/mosaic/content.go]: A better Golang convention is to use `BackgroundURL`
And for the JSON name field, I would choose `background_url`

**Review by mauricioabreu — CHANGES_REQUESTED** [review]
A few changes and we are good to go. Good job!

File: internal/mosaic/content.go (hunk 1 above)
Comment (luiscovelo) [reply internal/mosaic/content.go]: Alright, i wasn't attentive of this.

Fixed.

**Review by mauricioabreu — APPROVED** [review]
Good job!

**Pull request merged**